### PR TITLE
Random small tweaks

### PR DESCRIPTION
**Pull request by tomaszmrugalski**
This MR contains a collection of random small tweaks done, when doing clean reinstall after `svarog` rename.

### CODE REVIEW

**Comment by tomaszmrugalski** [comment]
This PR fixes setup problems I've encountered on both the server and client side. Most of them are minor tweaks, except the station/setup.py, which had two problems. First, it wasn't handling the analog-signal-estimator with its custom download path properly. Second, it didn't set up the alias (see #13) for details.

**Comment by tomaszmrugalski** [comment]
This addresses #13.

File: station/config.yml.template [hunk 1]
```diff
@@ -25,5 +25,5 @@ satellites:
 server:
   id: '1'
   secret: 0123456789abcdef0123456789abcdef
-  url: http://127.0.0.1:5000/receive
+  url: https://svarog.space/receive
```
Comment (fivitti) [inline station/config.yml.template:28]: Are you sure that it should be our server by default? It is easier for us, but new users probably will be test software locally first.

File: station/setup.py [hunk 2]
```diff
@@ -5,31 +5,115 @@
 import stat
 import sys
 
-REQUIREMENTS = [i.strip() for i in open("requirements.txt").readlines()]
+
+def get_requirements_and_links():
```
Comment (fivitti) [inline station/setup.py:9]: The ```setup.py``` is a standard Python way to install and publish packages.
I notice that in our case it has more disadvantages than advantages as not all our dependencies are in PyPI. Few of them aren't even Python scripts.
I will read a little about how to do it better.

**Review by fivitti — COMMENTED** [review]
Good job! I hope that it is the last rename.

File: station/config.yml.template (hunk 1 above)
Comment (tomaszmrugalski) [reply station/config.yml.template:28]: We're not exactly having a problem with too many users right now. I'd open a bottle of champagne if there's anyone except us submitting any observations. Ok, we could keep it as 127.0.0.1:5000, but the proper URL should be there commented out. Would that work?

File: station/setup.py (hunk 2 above)
Comment (tomaszmrugalski) [reply station/setup.py:9]: I think there's a trade off between turning this into everything-included-with-a-kitchen-sink. I do like the detection of what is installed. Perhaps it should do what brew does on Mac? Prints extra steps necessary to complete, but let the user do it?

Personally, I would focus more on actual reception capabilities. This is never going to be fully automated installation experience. "Some assembly required" will always be part of it.

**Pull request merged**